src/nbl/ui/CWindowWin32.cpp

```diff
@@ -384,72 +384,65 @@ namespace ui
 				if ((rawMouse.usFlags & MOUSE_MOVE_RELATIVE) == MOUSE_MOVE_RELATIVE && (rawMouse.lLastX != 0 || rawMouse.lLastY != 0))
```

Comment:
I dont get what's happening in this if-statement

Reply:
Yeah this is another thing i should check, but in theory this `if` checks if the mouse movement is sent relative to the previous mouse position and it's not (0, 0)

Reply:
why would you get a relative movement event with zero movement XD?

Reply:
Yeah that does make sense XD

Reply:
> Yeah that does not make sense XD

Reply:
I mean you can assert that the movement is not 0, and we'll see if its possible to get such weird events